js/gulpfile.js

```diff
@@ -26,6 +26,13 @@ const phpunitConfig = __dirname + '/../phpunit.xml';
 const karmaConfig = __dirname + '/karma.conf.js';
 const destinationFolder = __dirname + '/build/';
 const sources = [
+    'node_modules/es6-shim/es6-shim.min.js',
```

Comment:
btw, I think we can remove es6-shim now that all browsers support es6 for quite some time